cmd/deploy_delete.go

```diff
@@ -23,8 +23,15 @@ func newDeleteDeploymentCmd(deployConfig *deployCommandConfig) *cobra.Command {
 	var deployConfigFile string
 	var deleteDeploymentCmd = &cobra.Command{
 		Use:   "delete",
-		Short: "Delete your deployed Appsody project from a Kubernetes cluster",
-		Long:  `This command deletes your deployed Appsody project from the configured Kubernetes cluster using your existing deployment manifest.`,
+		Short: "Delete your deployed Appsody project from a Kubernetes cluster.",
+		Long: `Delete your deployed Appsody project from the configured Kubernetes cluster using your existing deployment manifest.
+
+By default, the command looks for the deployed project in the "default" namespace, and uses the generated "app-deploy.yaml" deployment manifest, unless you specify otherwise.`,
```

Comment:
I'd remove the comma in the middle, to clarify that "unless you specify otherwise" applies to both the namespace and the deployment manifest:
"By default, the command looks for the deployed project in the "default" namespace and uses the generated "app-deploy.yaml" deployment manifest, unless you specify otherwise."

Reply:
According to the docs, you have to run this command from your Appsody project directory, so that would be worth adding to the help?

Reply:
I have added this now, thanks